### PR TITLE
[VnV Plan] Add System Tests for Work Orders

### PR DESCRIPTION
**Pull request by arkinmodi**
Close https://github.com/arkinmodi/project-sayyara/issues/180

### CODE REVIEW

File: docs/VnVPlan/VnVPlan.tex [hunk 1]
```diff
@@ -188,50 +189,138 @@ \subsection{Tests for Functional Requirements}
...
+	\item \textbf{FRT-BE17-1}
 
-	Control: Manual versus Automatic
+	      Control: Manual
 
-	Initial State:
+	      Initial State: Shop Owner/Employee Account with a registered shop and a quote
 
-	Input:
+	      Input: A new appointment created at the shop with a quote ID
```
Comment (LeonSo7) [inline docs/VnVPlan/VnVPlan.tex:205]: Is the ID an input if it is automatically generated by the system? I think we said the system would do this right?

File: docs/VnVPlan/VnVPlan.tex [hunk 2]
```diff
@@ -188,50 +189,138 @@ \subsection{Tests for Functional Requirements}
...
+	\item \textbf{FRT-BE18}
 
-\subsubsection{Area of Testing2}
+	      Control: Manual
 
-...
+	      Initial State: Shop Owner/Employee Account with a registered shop, an upcoming appointment, and a
+	      work order associated to the appointment
+
+	      Input: Appointment is deleted by appointment ID
```
Comment (LeonSo7) [inline docs/VnVPlan/VnVPlan.tex:235]: Same here, will the system create the ID for appointments?

File: docs/VnVPlan/VnVPlan.tex (hunk 1 above)
Comment (arkinmodi) [reply docs/VnVPlan/VnVPlan.tex:205]: I think what I was trying to say is that the appointment is linked to the quote.

File: docs/VnVPlan/VnVPlan.tex (hunk 2 above)
Comment (arkinmodi) [reply docs/VnVPlan/VnVPlan.tex:235]: This one is for deleting the appointment so we would need the appointment ID to know which on to delete. Since this is now manual I can reword it to say that it is deleted from the UI.

**Review by LeonSo7 — APPROVED** [review]
